src/app/navbar/navbar.component.html

```diff
@@ -8,6 +8,45 @@
 
     <div class="navbar-collapse collapse">
       <app-search-bar></app-search-bar>
+      <form class="navbar-form navbar-left">
+        <div class="input-group" id="nav-group">
+          <input type="text" name="query" class="form-control" id="nav-input" placeholder="Search" (keyup.enter)="submit()" [(ngModel)]="searchdata.query">
```

Comment:
@Marauderer97 since this pull is already merged , make an other pull removing these lines. I somehow missed this while reviewing the pull.